app-analytics-hub-android/library/maven-push.gradle

```
@@ -23,12 +23,12 @@ def isReleaseBuild() {
 
 def getReleaseRepositoryUrl() {
     return hasProperty('RELEASE_REPOSITORY_URL') ? RELEASE_REPOSITORY_URL
-            : "https://oss.sonatype.org/service/local/staging/deploy/maven2/"
+            : "https://aws.oss.sonatype.org/service/local/staging/deploy/maven2/"
 }
 
 def getSnapshotRepositoryUrl() {
     return hasProperty('SNAPSHOT_REPOSITORY_URL') ? SNAPSHOT_REPOSITORY_URL
-            : "https://oss.sonatype.org/content/repositories/snapshots/"
+            : "https://aws.oss.sonatype.org/content/repositories/snapshots/"
```

Is `SNAPSHOT_REPOSITORY_URL` coming from gradle.properties?

If it is, we should fail the task here if it's not present with a useful error instead of duplicating values. Same for `getReleaseRepository`.

---

Throwing an error for snapshot url, release url, username and password.